#! /bin/sh /usr/share/dpatch/dpatch-run
## ocaml-interp_manpage.dpatch by Ralf Treinen <treinen@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: typo in ocaml manpage. <treinen@debian.org>

@DPATCH@
diff -urNad ocaml-3.10.0~beta~/man/ocaml.m ocaml-3.10.0~beta/man/ocaml.m
--- ocaml-3.10.0~beta~/man/ocaml.m	2002-02-08 11:16:10.000000000 +0100
+++ ocaml-3.10.0~beta/man/ocaml.m	2007-04-11 17:51:50.000000000 +0200
@@ -25,7 +25,7 @@
 command is the toplevel system for Objective Caml,
 that permits interactive use of the Objective Caml system through a
 read-eval-print loop. In this mode, the system repeatedly reads Caml
-phrases from the input, then typechecks, compile and evaluate
+phrases from the input, then typechecks, compiles and evaluates
 them, then prints the inferred type and result value, if any. The
 system prints a # (sharp) prompt before reading each phrase.
 
